api/src/pcapi/routes/serialization/offerers_serialize.py

```diff
@@ -37,8 +37,11 @@ class GetOffererVenueResponseModel(BaseModel, AccessibilityComplianceMixin):
     _humanize_venue_label_id = humanize_field("venueLabelId")
 
     @classmethod
+    # pylint: disable=dangerous-default-value
     def from_orm(
-        cls, venue: offerers_models.Venue, venues_with_offers: Iterable[int] = None
+        cls,
+        venue: offerers_models.Venue,
+        venues_with_offers: Iterable[int] = [],
```

Comment:
Réponse à la  suggestion de Victor : non, car on aimerait bien ne pas passer `None`. :) Mais on peut faire ça : 

```diff
- venues_with_offers: Iterable[int] = [],
+ venues_with_offers: Iterable[int] = (),
```
C'est itérable, c'est vide et ça n'est pas une "dangerous default value".